src/python/pants/backend/jvm/tasks/bootstrap_jvm_tools.py

```diff
@@ -189,6 +194,15 @@ def execute(self):
         dep_spec = jvm_tool.dep_spec(self.context.options)
         callback = self.cached_bootstrap_classpath_callback(dep_spec, jvm_tool)
         callback_product_map[jvm_tool.scope][jvm_tool.key] = callback
+      if self.get_options().eager:
+        for scope, callbacks_by_key in callback_product_map.items():
+          for key, callback in callbacks_by_key.items():
+            self.context.log.info('Eagerly resolving {} for scope {}.'.format(key, scope))
```

Comment:
Rather than including `eager` in the log message, maybe put eager resolution under a workunit? Would have the advantage of making it easier to track.

Reply:
I.e., a single workunit that contains all the resolution workunits? That works.

Reply:
Done!